### PR TITLE
Add tool argNorm

### PR DESCRIPTION
**Pull request by hugolefeuvre**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [x] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

**Review by bgruening — COMMENTED** [review]
Cool, quick review inline. I have just ponited out the `argument=""` feature. https://docs.galaxyproject.org/en/latest/dev/schema.html#id41 maybe that is interesting for you.


Thanks a lot!

File: tools/argnorm/argnorm.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<tool id="argnorm" name="argNorm" version="@VERSION@+galaxy0" profile="22.05">
+    <description>a tool to normalize antibiotic resistance genes (ARGs) by mapping them to the antibiotic resistance ontology (ARO)</description>
+    <macros>
+        <token name="@VERSION@">0.6.0</token>
+    </macros>
+    <xrefs>
+       <xref type="bio.tools">argnorm</xref>
```
[the remark this answers is not in the document]
Comment (hugolefeuvre) [reply tools/argnorm/argnorm.xml:7]: I don't know, is it useful ? Is it why the linting get in error ?

[the remark this answers is not in the document]
Comment (bgruening) [reply tools/argnorm/argnorm.xml:7]: Yes, this is why the linting fails. If we have a proper bio.tools enty, Galaxy can do some additional things regarding Finability and such ... We recommend it, but if its too much effort remove this tag.

[the remark this answers is not in the document]
Comment (hugolefeuvre) [reply tools/argnorm/argnorm.xml:7]: Ok, I'll give it a try and if I don't make it I'll remove the tag.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks a lot!

**Pull request merged**